### DISCUSSION TITLE
How do you "clear history"?

### DISCUSSION THREAD

**Discussion by brandon-9** [spatie/ray#608] [Q&A] [answered]
Simple question. Every now and then I'd like to just clear the past logs in Ray. How do I do that?
Also, possible to reverse the order so most recent logs are at the top?

**Comment by Polfo** [comment, marked as answer]
ctrl-shift-K

**Reply by freekmurze** [reply]
> Also, possible to reverse the order so most recent logs are at the top?

Unfortunately, this is not possible, because of the specific high performance renderer that is used under the hood.
